### ISSUE TITLE
call `deployContracts` only in tests

### ISSUE THREAD

**Issue by jellegerbrandy** [Paratii-Video/paratii-portal#36]
https://github.com/Paratii-Video/paratii-portal/blob/16-player/src/scripts/utils/ParatiiLib.js called "deployContracts".

the "deplloyContrcts" function is really a kind of sysadmin utility. In normal usage, all contracts are deployed on the blockchain (by us), and clients use those given contracts. The only thing we give the client is the address of the `ParatiiRegistry`  conract, which contains references to the address of all other contrcts.

In other words, the client should _never_ call depoyContracts. This call to "deployContracts" should move to the tests setup.

**Comment by jellegerbrandy** [comment]
Here's a corresponding issue for mking this clearer in the API: https://github.com/Paratii-Video/paratii-lib/issues/48
